master/internal/experiment.go

```diff
@@ -280,6 +281,11 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 	case job.GetJobSummary:
 		ctx.Respond(e.toV1Job().Summary)
 
+	case job.SetRP:
+		if err := e.setRP(ctx, msg); err != nil {
+			return err
```

Comment:
returning an error I think will kill the actor. do we want this? when setting priority and the like we don't do this.

Reply:
we should only return error if there is a database issue - otherwise we do a ctx.Response from inside the setRP function

Reply:
fixed!